src/supermin-init-prelude.sh

```diff
@@ -30,7 +30,10 @@ if [ -L "${workdir}"/src/config ]; then
     mount -t 9p -o rw,trans=virtio,version=9p2000.L source "${workdir}"/src/config
 fi
 mkdir -p "${workdir}"/cache /host/container-work
-mount /dev/sdb1 "${workdir}"/cache
+# XXX: should probably be label-based
```

Comment:
labels require some sort of udev to create the `/dev/disk/by-[part]label`, in the interest of simplicity I'm fine leaving this as is.